src/pkg/services/m365/api/mail.go

```diff
@@ -444,6 +444,30 @@ func (c Mail) PostItem(
 	return itm, nil
 }
 
+func (c Mail) MoveItem(
+	ctx context.Context,
+	userID, oldContainerID, itemID, newContainerID string,
```

Comment:
My ocd is undecided between "move this whole thing to that container" vs stepping parameters in order as they would appear in the hierarchy.  Either way works, but figured I'd put up the nit.
```suggestion
	userID, oldContainerID, newContainerID, itemID string,
```